backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java

```diff
@@ -1600,7 +1600,8 @@ private DocStatus completeIt0()
 		//
 		// Offers
 		if (X_C_DocType.DOCSUBTYPE_Proposal.equals(docSubType)
-				|| X_C_DocType.DOCSUBTYPE_Quotation.equals(docSubType))
+				|| X_C_DocType.DOCSUBTYPE_Quotation.equals(docSubType)
+				|| X_C_DocType.DOCSUBTYPE_Rahmenvertrag.equals(docSubType))
```

Comment:
please chreate the new doctypes such that we have here there EN names, (like "Quotation")

Reply:
So, do you have in here the SQL to deactivate the doc-types? I didn't see it, but in this large SQL file i might have overlooked it

Reply:
Yep, I've changed the script to create the docTypes with IsActive='N'

Reply:
>please chreate the new doctypes such that we have here there EN names, (like "Quotation")

Done.